spec/event_store_client/data_decryptor_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'securerandom'
+require 'support/dummy_repository'
+
+module EventStoreClient
+  RSpec.describe DataDecryptor do
+    describe '#call' do
+      let(:repository) { DummyRepository.new }
+
+      subject { described_class.new(data: data, schema: schema, repository: repository) }
+
+      it 'returns decrypted data' do
+        expect(subject.call).to eq(
+          user_id: user_id,
+          first_name: 'Anakin',
+          last_name: 'Skylwalker',
+          profession: 'Jedi'
+        )
```

Comment:
You can probably remove DRY by extracting above expected data to the let like:
let(:expected_data) { {user_id: user_id, first_name: ...... } }
and just expect(subject.call).to eq(expected_data)